cfme/intelligence/reports/__init__.py

```diff
@@ -7,6 +7,7 @@
 
 from cfme.base import Server
 from cfme.base.login import BaseLoggedInPage
+from cfme.utils import version
```

Comment:
**REQUIRED**
It's better to use `appliance.version` and `self.browser.product_version` instead of `cfme.utils.version`